lib/rack/proxy.rb

```diff
@@ -42,8 +42,9 @@ def reconstruct_header_name(name)
     def initialize(opts = {})
       @streaming = opts.fetch(:streaming, true)
       @ssl_verify_none = opts.fetch(:ssl_verify_none, false)
-      @backend = URI(opts[:backend]) if opts[:backend]
+      @backend = opts[:backend] ? URI(opts[:backend]) : nil
```

Comment:
```
.rack-proxy/lib/rack/proxy.rb:89: warning: instance variable @backend not initialized
rack-proxy/lib/rack/proxy.rb:101: warning: instance variable @ssl_version not initialized
```
Depending on the test, some tests initialize neither, one, or both but the tests pass with either coding style